ui/raidboss/data/07-dt/map/cenote_ja_ja_gural.ts

```diff
@@ -0,0 +1,108 @@
+import Conditions from '../../../../../resources/conditions';
+import { Responses } from '../../../../../resources/responses';
+import ZoneId from '../../../../../resources/zone_id';
+import { RaidbossData } from '../../../../../types/data';
+import { TriggerSet } from '../../../../../types/trigger';
+
+const cenoteJaJaOutputStrings = {
+  spawn: {
+    en: '${name} spawned!',
+    de: '${name} erscheint!',
+    fr: '${name} apparait !',
+    ja: '${name} 現れる！',
+    cn: '已生成 ${name}!',
+    ko: '${name} 등장!',
+  },
+} as const;
+
+export type Data = RaidbossData;
+
+const triggerSet: TriggerSet<Data> = {
+  id: 'CenoteJaJaGural',
+  zoneId: ZoneId.CenoteJaJaGural,
+
+  triggers: [
+    // ---------------- random treasure mobs ----------------
+    {
+      id: 'Cenote Ja Ja Alpaca of Fortune Spawn',
+      // 13240 = Alpaca of Fortune
+      type: 'AddedCombatant',
+      netRegex: { npcNameId: '13240' },
+      suppressSeconds: 1,
+      infoText: (_data, matches, output) => output.spawn!({ name: matches.name }),
+      outputStrings: {
+        spawn: cenoteJaJaOutputStrings.spawn,
+      },
+    },
+    {
+      id: 'Cenote Ja Ja Uolon of Fortune Spawn',
+      // 13241 = Uolon of Fortune
+      type: 'AddedCombatant',
+      netRegex: { npcNameId: '13241' },
+      suppressSeconds: 1,
+      infoText: (_data, matches, output) => output.spawn!({ name: matches.name }),
+      outputStrings: {
+        spawn: cenoteJaJaOutputStrings.spawn,
+      },
+    },
+    {
+      id: 'Cenote Ja Ja Tuligora Troupes Spawn',
+      // 13242 = Turali Onion
+      // 13243 = Turali Eggplant
+      // 13244 = Turali Garlic
+      // 13245 = Turali Tomato
+      // 13246 = Tuligora Queen
+      type: 'AddedCombatant',
+      netRegex: { npcNameId: '1324[2-6]', capture: false },
```

Comment:
I don't remember the rationale (maybe @JLGarber does?), but I think we've been trying not to use regex character classes as values, and instead doing arrays, e.g.:
`const tuligoraTroupesIds = [13242, 13243, 13244, 13245, 13246];`
and then
`netRegex: { npcNameId: tuligoraTroupesIds, capture: false }`